api/routes/home.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express');
+const router = express.Router();
+
+router.post('/hello', (req, res, next) => {
```

Comment:
Ideally your routes should be more descriptive than "hello". Maybe something like "echo" would be more appropriate here

Reply:
I understand. I will try to write more readable code and comment too next time.